samples/objc/FirebaseUI-demo-objc/Samples/Auth/FUIAuthViewController.m

```diff
@@ -426,7 +426,7 @@ + (NSArray *)getListOfIDPs:(NSArray<NSIndexPath *> *)selectedRows
           break;
         case kIDPFacebook:
           provider = useCustomScopes ? [[FUIFacebookAuth alloc] initWithAuthUI:[FUIAuth defaultAuthUI]
-                                                                   permissions:@[@"email",
+                                                                   permissions:@[@"password",
```

Comment:
This one is not a provider and should remain email.

Reply:
Nice! Done